RecoPixelVertexing/PixelTriplets/src/CAHitQuadrupletGenerator.cc

```diff
@@ -54,13 +57,21 @@ caHardPtCut(cfg.getParameter<double>("CAHardPtCut"))
 }
 
 void CAHitQuadrupletGenerator::fillDescriptions(edm::ParameterSetDescription& desc) {
+  edm::ParameterSetDescription empty;
+  empty.setAllowAnything();
+
   desc.add<double>("extraHitRPhitolerance", 0.1);
   desc.add<bool>("fitFastCircle", false);
   desc.add<bool>("fitFastCircleChi2Cut", false);
   desc.add<bool>("useBendingCorrection", false);
   desc.add<double>("CAThetaCut", 0.00125);
   desc.add<double>("CAPhiCut", 10);
   desc.add<double>("CAHardPtCut", 0);
+  desc.add<std::vector<std::string>>("layerList",{});
+  desc.add<edm::ParameterSetDescription>("BPix", empty);
+  desc.add<edm::ParameterSetDescription>("FPix", empty);
+  desc.add<bool>("isFastSim", false);
+  desc.add<std::vector<unsigned>>("layerPairs", std::vector<unsigned>{0});
```

Comment:
I am actually trying to add these parameters to SeedFinderSelector (see commit https://github.com/angirar/cmssw/pull/3/commits/5f0323a02b86bd680f57d736baf7c5da9edabf6e ) but somehow it gives error everytime as - KeyError: 'Unknown parameter name xxxx specified while calling Modifier'

Reply:
You get much more information from python errors by running the config with `python config.py` instead of `cmsRun config.py`. Could you share a full trace from `python`?